operator/roles/kiali-deploy/tasks/openshift/os-main.yml

```diff
@@ -65,5 +65,9 @@
 - name: Process OpenShift Console Links
   k8s:
     definition: "{{ lookup('template', 'templates/openshift/console-links.yaml') }}"
+  vars:
+    # For some reason, in some cases when the operator runs, the kiali.io/member-of tag is not set yet.
+    # Maistra-ones are always present however.
```

Comment:
Of course, this requires you to be running with ServiceMesh installed. Which, again, is why I say if you are going to assume Maistra is installed, then don't worry about any of this since when running with ServiceMesh you will NEVER get accessible_namespaces set to `**`.